master/security/high-connection-workloads.md

```diff
@@ -0,0 +1,93 @@
+---
+title: Enable extreme high-connection workloads 
+---
+
+### Big Picture
+
+Use a Calico network policy rule to bypass Linux conntrack for traffic to extreme high-connection workloads.
```

Comment:
Might just be me, but I find "extreme high connection workloads" to be a somewhat awkward or confusing term.

How do you feel about something like "for traffic to workloads which serve an extremely high rate of connections" or "for workloads which serve an extremely large number of connections"?

Maybe just "extremely high-connection workloads"?

Reply:
I agree with this. Alex overruled my other suggestions. Let me try to persuade him to take a look again. Usually, we don't use hyper language like "extremely high" when high already indicates something above and beyond. So I need convincing that in conntrack tables people make some distinction in their minds between "high" and "extremely high".

Reply:
How about this:
Use a Calico network policy rule to bypass Linux conntrack for traffic to workloads that experience extremely large number of connections.

Reply:
Fixed